examples/ansible/setup_keylime.yml

```diff
@@ -0,0 +1,49 @@
+---
+# Ansible Playbook: Setup Keylime on SUSE ALP Dolomite
+# Description: This Ansible playbook automates the deployment and setup of Keylime verifier, registrar and tenant on a SUSE ALP Dolomite host
+# The steps encompass fetching the Keylime image from the specified registry, creating a necessary volume for the Keylime control plane,
+# launching the Keylime container along with related services, and executing the tenant CLI to interact with Keylime.
+# Documentation reference: [https://documentation.suse.com/alp/dolomite/html/alp-dolomite/keylime-remote-attestation.html#keylime-run-with-podman]
+
+- name: Setup Keylime verifier, registrar and tenant
+  hosts: alphost
+  become: true
+  vars:
+    workload:
+      name: keylime
+      image: registry.opensuse.org/devel/microos/containers/containerfile/opensuse/keylime-control-plane:latest
+
+  tasks:
+    - name: Retrieve images for Keylime
+      containers.podman.podman_image:
+        name: "{{ workload.image }}"
+        state: present
+
+    - name: Create the keylime-control-plane volume
+      containers.podman.podman_volume:
+        name: keylime-control-plane-volume
+        state: present
+
+    - name: Start the Keylime container and related services
+      ansible.builtin.command: >-
+        podman container runlabel run "{{ workload.image }}"
+      register: workload_runlabel_run
+      changed_when:
+        - workload_runlabel_run.stdout is regex("^[a-f0-9]{64}$")
+
+    - name: Pause for 5 seconds
+      ansible.builtin.pause:
+        seconds: 5
+
+    - name: Executing the tenant CLI
+      ansible.builtin.command: >-
+        podman run --rm -v keylime-control-plane-volume:/var/lib/keylime/ keylime-control-plane:latest
```

Comment:
Consider using a `--` argument to delineate the end of the podman arguments/options and the beginning of the keylime_tenant arguments...

Reply:
Podman seems to be interpreting -- incorrectly.

podman run --rm \
> -v keylime-control-plane-volume:/var/lib/keylime/ \
> keylime-control-plane:latest \
> -- keylime_tenant -v 10.88.0.1 -r 10.88.0.1 --cert default -c reglist
Error: runc: runc create failed: unable to start container process: exec: "--": executable file not found in $PATH: OCI runtime attempted to invoke a command that was not found